papers/p1729r4.bs

```diff
@@ -1622,6 +1623,16 @@ Header `<scan>` synopsis {#spec-header-synopsis}
 #include &lt;ranges&gt;
 
 namespace std {
+  template&lt;class charT, class... Args&gt;
+    struct basic_scan_format_string;
```

Comment:
In scnlib this type is actually
```cpp
template <typename CharT, typename Source, typename... Args>
struct basic_scan_format_string;
```

That's because whether an argument `A` is scannable can also depend on the source: `string_view`s are not scannable from a non-contiguous range. It could be possible to get this diagnostic at compile-time another way, but it may not be as elegant.

Reply:
Makes sense. Let's fix this separately.